posthog/tasks/periodic_digest.py

```diff
@@ -285,3 +289,47 @@ def send_all_periodic_digest_reports(
     except Exception as err:
         capture_exception(err)
         raise
+
+
+def send_digest_notifications(
+    *,
+    team_id: int,
+    organization_id: Optional[str],
+    event_name: str,
+    properties: dict[str, Any],
+    notification_type: NotificationSetting,
+    timestamp: Optional[datetime] = None,
+) -> None:
+    """
+    Determines eligible recipients and sends individual notifications for digest reports.
+    """
+    pha_client = Client("sTMFPsFhdP1Ssg")
```

Comment:
So bad we are hard coding this 😑

Reply:
I think we hardcode it so that we still get the intel from self-hosted projects. If it were an env var then it'd go elsewhere.